Tools/Pipeline/Windows/Controls/FilterOutputControl.cs

```diff
@@ -0,0 +1,138 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.IO;
+using System.Windows.Forms;
+
+namespace MonoGame.Tools.Pipeline.Windows.Controls
+{
+    public partial class FilterOutputControl : TreeView
+    {
+        const int WM_VSCROLL = 0x0115;
+        const int SB_BOTTOM  = 0x07;
+        [System.Runtime.InteropServices.DllImport("user32.dll", CharSet = System.Runtime.InteropServices.CharSet.Auto, SetLastError = false)]
+        static extern IntPtr SendMessage(IntPtr hWnd, uint Msg, int wParam, int lParam);
+
+        BuildIcons _buildIcons;
+        TreeNode tmpIter;
+        
+        Uri folderUri;
+        Uri outputUri;
+
+        public FilterOutputControl(): base()
+        {
+            this._buildIcons = new BuildIcons();
+            this.ImageList = _buildIcons.Icons;
+        }
+        
+        
+        internal void SetBaseFolder(IController controller)
+        {
+            string pl = ((PipelineController)controller).ProjectLocation;
+            if (!pl.EndsWith(System.IO.Path.DirectorySeparatorChar.ToString()))
+                pl += System.IO.Path.DirectorySeparatorChar;
+            folderUri = new Uri(pl);
+
+            string pod = ((PipelineController)controller).ProjectOutputDir;
+            if (!pod.EndsWith(System.IO.Path.DirectorySeparatorChar.ToString()))
+                pod += System.IO.Path.DirectorySeparatorChar;
+            pod = Path.Combine(pl, pod);
+            outputUri = new Uri(pod);
+        }
+
+
+        internal void AppendText(string text)
```

Comment:
See the screenshot how it says, 
Cleaning GeometryA.xnb
Building GeometryA.xnb
For doing this I had to expose the project.OutputDir from pipelineController.
In your case the first line will be something like:
Cleaning \bin\Linux\GeometryA.xnb
Similarly I had a project where the output is a couple of folders down and then a few folders up, and the output was like
Cleaning ../../../Game/projCodename/projCodename.WindowsPhone/Content/GeometryA.fbx
This goes again the clean output you were going for.